From 3d493245c9f10cfc6c993bfd10a79d450ee5d99c Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Sat, 13 Sep 2014 17:12:43 +0200
Subject: [PATCH 24/42] first_run: deactivate autoupdate globally

We do not have a tunable in chrome://settings (or similar) to
turn it back on.
---
 chrome/browser/extensions/extension_system_impl.cc | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/chrome/browser/extensions/extension_system_impl.cc b/chrome/browser/extensions/extension_system_impl.cc
index d1d5e12..937d109 100644
--- a/chrome/browser/extensions/extension_system_impl.cc
+++ b/chrome/browser/extensions/extension_system_impl.cc
@@ -310,6 +310,9 @@ void ExtensionSystemImpl::Shared::Init(bool extensions_enabled) {
   runtime_data_.reset(new RuntimeData(ExtensionRegistry::Get(profile_)));
 
   bool autoupdate_enabled = !profile_->IsGuestSession();
+  if (autoupdate_enabled)
+    fprintf(stderr, "*** autoupdate was enabled, overriding with false\n");
+  autoupdate_enabled = false;
 #if defined(OS_CHROMEOS)
   if (!extensions_enabled)
     autoupdate_enabled = false;
-- 
2.4.0

